Vagrantfile

```diff
@@ -58,5 +59,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     }
     ansible.extra_vars = { "islandora_distro" => $vagrantBox }
   end
-
+  unless  $reset_roles.eql? "FALSE"
+    # Fires last to modify one last change.
```

Comment:
If this fires last, wouldn't it delete after the ansible-galaxy command? Because that is the one that populates the roles/external with the external roles.

Reply:
Yes, and no. It shouldn't fire at all unless you're explicitly trying to run it before you rebuild. Assuming you have already ran vagrant up and need to wipe out your external roles before rebuilding, you could run the `ISLANDORA_VAGRANT_REST_ROLES='TRUE' vagrant reload` then destroy and up. I know we could easily just delete the folder ourselves but this automated the process for people that are new at it.

Reply:
The no part was, No it shouldn't ever fire unless you specify it in the provisions part. And if you run it as suggested it should be the only provision to fire regardless.